<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Publications - Joel F Swift</title>
  <meta name="description" content="Academic webpage of Dr. Joel F Swift">
  <link rel="stylesheet" href="/JoelFSwift.github.io/assets/main.css">
  <link rel="canonical" href="https://kenizzer.github.io/JoelFSwift.github.io/publications/">
  <link rel="shortcut icon" type ="image/x-icon" href="https://kenizzer.github.io/JoelFSwift.github.io/faviconJFS.ico">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">

  <link rel="preconnect" href="https://player.vimeo.com">
  <link rel="preconnect" href="https://i.vimeocdn.com">
  <link rel="preconnect" href="https://f.vimeocdn.com">



<!-- Google Analytics (original) -->
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', '', 'auto');
  ga('send', 'pageview');

</script>

<!-- Global site tag (gtag.js) - Google Analytics 4 -->
<script async src="https://www.googletagmanager.com/gtag/js?id="></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', '');
</script>

<!-- Google Tag Manager -->
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
    })(window,document,'script','dataLayer','');</script>
<!-- End Google Tag Manager -->



<script>
MathJax = {
    tex: {
    inlineMath: [ ['$', '$'], ['\\(', '\\)'] ],
    tags: 'ams'  // should be 'ams', 'none', or 'all' }. This line makes the equation numbering and labeling work
    }, 
    svg: {
    fontCache: 'global'
    }
};
</script>
<script
    type="text/javascript" id="MathJax-script" async
    src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js">
</script> 

</head>


  <body>

    <!-- Google Tag Manager (noscript) -->
<noscript><iframe src="https://www.googletagmanager.com/ns.html?id="
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<!-- End Google Tag Manager (noscript) -->

<nav class="navbar sticky-top navbar-expand-md navbar-dark bg-dark">
    <a class="navbar-brand" href="https://kenizzer.github.io/JoelFSwift.github.io/">
     <img src="https://kenizzer.github.io/JoelFSwift.github.io/faviconJFS.ico" width="30" height="30" style="margin-right:5px" class="d-inline-block align-top" alt="">
      Joel F Swift
    </a>
    <button class="toggler navbar-toggler collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#navbarColor02" aria-controls="navbarColor02" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarColor02">
        <div class="navbar-nav mr-auto">
            <a class="nav-item nav-link" href="https://kenizzer.github.io/JoelFSwift.github.io/">Home</a>
                   
                <a class="nav-item nav-link" href="https://kenizzer.github.io/JoelFSwift.github.io/about">About</a>
                   
                <a class="nav-item nav-link" href="https://kenizzer.github.io/JoelFSwift.github.io/publications">Publications</a>
                   
                <a class="nav-item nav-link" href="https://kenizzer.github.io/JoelFSwift.github.io/research">Research</a>
                   
                <a class="nav-item nav-link" href="https://kenizzer.github.io/JoelFSwift.github.io/microscopy">Microscopy</a>
            
        </div>
    </div>
</nav>


    <div class="container-fluid">
      <div class="row">
        <div id="gridid" class="col-sm-12 col-xs-12">
  <style>
.jumbotron{
    padding:3%;
    padding-bottom:10px;
    padding-top:10px;
    margin-top:10px;
    margin-bottom:30px;
}
</style>

<div class="jumbotron">
  <h3 id="preprints">Preprints</h3>
  <ol class="bibliography" reversed="reversed"><li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="jones2025rapid">Jones, D. H., Baca Cabrera, J. C., Behrend, D., Wells, D. M., Swift, J. F., Atkinson, J. A., Lobet, G., Hanlon, M. T., &amp; Schneider, H. M. (2025). The Rapid Anatomics Tool (RAT): A low-cost root anatomical phenotyping pipeline reveals changes in root anatomy along the root axis. In <i>BioRxiv</i>.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="jones2025rapid">Jones, D. H., Baca Cabrera, J. C., Behrend, D., Wells, D. M., Swift, J. F., Atkinson, J. A., Lobet, G., Hanlon, M. T., &amp; Schneider, H. M. (2025). The Rapid Anatomics Tool (RAT): A low-cost root anatomical phenotyping pipeline reveals changes in root anatomy along the root axis. In <i>BioRxiv</i>.</span></div> -->








<a href="https://arxiv.org/abs/10.1101/2025.09.05.674482" target="_blank"><button class="btn btn-primary btm-sm">ARXIV</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexjones2025rapid()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractjones2025rapid()">ABSTRACT</button>



<div id="ajones2025rapid" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@unpublished{jones2025rapid,
  title = {The Rapid Anatomics Tool (RAT): A low-cost root anatomical phenotyping pipeline reveals changes in root anatomy along the root axis},
  author = {Jones, Dylan H and Baca Cabrera, Juan C and Behrend, Dominik and Wells, Darren M and Swift, Joel F and Atkinson, Jonathan A and Lobet, Guillaume and Hanlon, Meredith T and Schneider, Hannah M},
  journal = {BioRxiv},
  year = {2025},
  doi = {10.1101/2025.09.05.674482}
}
</pre>
</div>


<div id="bjones2025rapid" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Root anatomical phenotyping has become a demonstrably essential part of investigating root physiology and in acquiring a holistic understanding of plant development. However, accessible high throughput methods for root anatomical analysis are still lacking. Here, we present the Rapid Anatomics Tool (RAT), a novel, low-cost platform for high throughput root anatomical imaging with a shallow learning curve for obtaining high quality images suitable for comparative analysis across a number of plant species. Its efficiency comes from combining blockface-like imaging and stain-free imaging using near-ultraviolet (nUV) autofluorescence utilising a combination of low-cost commercial equipment, readily available mechanical components, and custom designed and 3D printed tools. Using this platform, we investigated the anatomy of mature tissue along the axis of wheat crown roots, revealing a tendency of reduction in vascular complexity (expressed through a reduction in metaxylem number, area, and mean area per metaxylem file) from the basal to the distal region of the root. This study highlights the importance of thorough sampling strategies for investigating root anatomy in relation to organ function and introduces an accessible, relatively high-throughput platform to support such research.</pre>
</div>

<script>
function toggleBibtexjones2025rapid(parameter) {
    var x= document.getElementById('ajones2025rapid');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractjones2025rapid(parameter) {
    var x= document.getElementById('bjones2025rapid');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li></ol>
</div>

<div class="jumbotron">
  <h3 id="refereed-journal-articles">Refereed journal articles</h3>
  <ol class="bibliography" reversed="reversed"><li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="swift2025between">Swift, J. F., Thimesch, D., Bengfort, L., Asif, S., &amp; Wagner, M. R. (2025). Between Two Extremes: Tripsacum dactyloides Root Anatomical Responses to Drought and Waterlogging. <i>Plant Direct</i>, <i>9</i>(7).</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="swift2025between">Swift, J. F., Thimesch, D., Bengfort, L., Asif, S., &amp; Wagner, M. R. (2025). Between Two Extremes: Tripsacum dactyloides Root Anatomical Responses to Drought and Waterlogging. <i>Plant Direct</i>, <i>9</i>(7).</span></div> -->








<a href="http://doi.org/10.1002/pld3.70097" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexswift2025between()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractswift2025between()">ABSTRACT</button>



<div id="aswift2025between" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{swift2025between,
  title = {Between Two Extremes: Tripsacum dactyloides Root Anatomical Responses to Drought and Waterlogging},
  author = {Swift, Joel F and Thimesch, Desi and Bengfort, Lucas and Asif, Shahzaib and Wagner, Maggie R},
  journal = {Plant Direct},
  volume = {9},
  number = {7},
  year = {2025},
  doi = {10.1002/pld3.70097}
}
</pre>
</div>


<div id="bswift2025between" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Plant roots are the critical interface between plants, soil, and microorganisms, and respond dynamically to changes in water availability. Although anatomical adaptations of roots to water stress (e.g., the formation of root cortical aerenchyma) are well documented, it remains unclear whether these responses manifest along the length of individual roots under both water deficiency and water overabundance. We investigated the anatomical responses of Tripsacum dactyloides L. to both drought and waterlogging stress at high spatial resolution. Nodal roots were segmented into one-centimeter sections from the tip to the base, allowing us to pinpoint regions of maximal anatomical change. Both stressors overall increased the proportion of root cortical aerenchyma, but metaxylem responses differed: waterlogging increased the proportion of the stele that was occupied by metaxylem with fewer but larger vessels. Drought significantly increased root hair formation within two centimeters of the root tip. The most pronounced anatomical changes occurred 3–7cm from the root tip, where cortical cell density declined as aerenchyma expanded. These findings highlight spatial variation in root anatomical responses to water stress and provide a framework that can inform sampling protocols for various other data types where sampling effort is limiting (e.g., microbiome, transcriptome, proteome).</pre>
</div>

<script>
function toggleBibtexswift2025between(parameter) {
    var x= document.getElementById('aswift2025between');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractswift2025between(parameter) {
    var x= document.getElementById('bswift2025between');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="migicovsky2025terroir">Migicovsky, Z., Swift, J. F., Awale, M., Helget, Z., Klein, L. L., Pinkner, L., Woodhouse, K., Cousins, P., Fennell, A. Y., Miller, A. J., &amp; others. (2025). Terroir and rootstock effects on leaf shape in California Central Valley vineyards. <i>Plants, People, Planet</i>, <i>7</i>(4), 1110–1122.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="migicovsky2025terroir">Migicovsky, Z., Swift, J. F., Awale, M., Helget, Z., Klein, L. L., Pinkner, L., Woodhouse, K., Cousins, P., Fennell, A. Y., Miller, A. J., &amp; others. (2025). Terroir and rootstock effects on leaf shape in California Central Valley vineyards. <i>Plants, People, Planet</i>, <i>7</i>(4), 1110–1122.</span></div> -->








<a href="http://doi.org/10.1002/ppp3.10620" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexmigicovsky2025terroir()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractmigicovsky2025terroir()">ABSTRACT</button>



<div id="amigicovsky2025terroir" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{migicovsky2025terroir,
  title = {Terroir and rootstock effects on leaf shape in California Central Valley vineyards},
  author = {Migicovsky, Zoe and Swift, Joel F and Awale, Mani and Helget, Zachary and Klein, Laura L and Pinkner, Leah and Woodhouse, Karoline and Cousins, Peter and Fennell, Anne Y and Miller, Allison J and others},
  journal = {Plants, People, Planet},
  volume = {7},
  number = {4},
  pages = {1110--1122},
  year = {2025},
  doi = {10.1002/ppp3.10620}
}
</pre>
</div>


<div id="bmigicovsky2025terroir" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Embedded in a single leaf shape are the latent signatures of genetic, developmental, and environmental effects. In viticulture, choice of location and rootstock are important decisions that affect the performance and production of the shoot. We hypothesize that these effects influence plant morphology, as reflected in leaf shape. We sample 1879 leaves arising from scion and rootstock combinations from commercial vineyards in the Central Valley of California. Our design tests 20 pairwise contrasts between Cabernet Sauvignon and Chardonnay scions from San Joaquin, Merced, and Madera counties from vines grafted to Teleki 5C, 1103 Paulsen, and Freedom rootstocks. We quantify clear differences between Cabernet Sauvignon and Chardonnay leaves. However, we also detect a separate, statistically independent source of shape variance that affects both Cabernet Sauvignon and Chardonnay leaves similarly. We find that this other shape difference is associated with differences in rootstock and location. The shape difference that arises from rootstock and location affects the basal part of the leaf near the petiole, known as the petiolar sinus, and affects its closure. This shape effect is independent from previously described shape differences that arise from genetic, developmental, or size effects.</pre>
</div>

<script>
function toggleBibtexmigicovsky2025terroir(parameter) {
    var x= document.getElementById('amigicovsky2025terroir');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractmigicovsky2025terroir(parameter) {
    var x= document.getElementById('bmigicovsky2025terroir');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="kolp2025evidence">Kolp, M., Marcello, M., Holt, A., Rossi, K., Zurawski, C., Cancelliere, K., Telemeco, S., Swift, J. F., Purple, K., &amp; Faulkner, C. (2025). Evidence of canine intestinal parasites and associated fecal bacteria among urban and rural dog parks in central Appalachia US. <i>Veterinary Parasitology, Regional Studies and Reports</i>, <i>62</i>.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="kolp2025evidence">Kolp, M., Marcello, M., Holt, A., Rossi, K., Zurawski, C., Cancelliere, K., Telemeco, S., Swift, J. F., Purple, K., &amp; Faulkner, C. (2025). Evidence of canine intestinal parasites and associated fecal bacteria among urban and rural dog parks in central Appalachia US. <i>Veterinary Parasitology, Regional Studies and Reports</i>, <i>62</i>.</span></div> -->








<a href="http://doi.org/10.1016/j.vprsr.2025.101280" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexkolp2025evidence()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractkolp2025evidence()">ABSTRACT</button>



<div id="akolp2025evidence" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{kolp2025evidence,
  title = {Evidence of canine intestinal parasites and associated fecal bacteria among urban and rural dog parks in central Appalachia US},
  author = {Kolp, Matthew and Marcello, M and Holt, A and Rossi, K and Zurawski, C and Cancelliere, K and Telemeco, S and Swift, JF and Purple, K and Faulkner, C},
  journal = {Veterinary parasitology, regional studies and reports},
  volume = {62},
  year = {2025},
  doi = {10.1016/j.vprsr.2025.101280}
}
</pre>
</div>


<div id="bkolp2025evidence" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Public dog parks provide opportunities for exercise and socialization, benefiting both dogs and their owners. However, dog parks can serve as reservoirs for gastrointestinal (GI) parasitic helminths, posing risks to canine and public health. This study investigated the presence of three common GI parasites, hookworm (Ancylostoma spp.), roundworm (Toxocara canis), and whipworm (Trichuris vulpis), in abandoned fecal samples from nine dog parks in Central Appalachia, comparing rural and urban environments. We also examined the relationship between parasite presence and fecal bacterial community composition using 16S rRNA sequencing. A total of 346 fecal samples were collected during 43 visits to parks in Tennessee and Kentucky. Centrifugal flotation (1.2 spg) revealed evidence of parasitic infection in 26.9 % of samples, with a higher percentage found in samples from rural parks (40.7 % of 135 samples) compared to urban parks (18.0 % of 211 samples; p &lt; 0.0001). Hookworm was the most common parasite identified (18.8 %), with greater numbers of eggs in rural parks. Mixed infections were observed in 6.9 % of samples and found more often in rural dog parks. 16S rRNA sequencing indicated that bacterial diversity and community composition were influenced by park location and parasite presence. Rural parks exhibited more diverse bacterial communities and higher proportions of Actinomycetota, while urban parks showed higher levels of Fusobacteriota. Parasite-positive samples had greater bacterial diversity and were associated with specific taxa, such as Clostridia, linked to GI issues. Our findings highlight the increased potential of GI parasitic helminth contamination in rural dog parks and highlight the potential for parasites to alter canine gut microbiomes. These results emphasize the need for public health education on routine anthelmintic treatments to reduce zoonotic risks. The study further underscores the importance of a One Health approach to managing parasite transmission in communal canine spaces. Future work should explore the mechanisms underlying GI parasite-microbiome interactions and extend these findings to other geographic regions.</pre>
</div>

<script>
function toggleBibtexkolp2025evidence(parameter) {
    var x= document.getElementById('akolp2025evidence');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractkolp2025evidence(parameter) {
    var x= document.getElementById('bkolp2025evidence');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="swift2025drought">Swift, J. F., Kolp, M. R., Carmichael, A., Ford, N. E., Hansen, P. M., Sikes, B. A., Kleiner, M., &amp; Wagner, M. R. (2025). Drought stress homogenizes maize growth responses to diverse natural soil microbiomes. <i>Plant and Soil</i>, <i>509</i>(1), 181–199.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="swift2025drought">Swift, J. F., Kolp, M. R., Carmichael, A., Ford, N. E., Hansen, P. M., Sikes, B. A., Kleiner, M., &amp; Wagner, M. R. (2025). Drought stress homogenizes maize growth responses to diverse natural soil microbiomes. <i>Plant and Soil</i>, <i>509</i>(1), 181–199.</span></div> -->








<a href="http://doi.org/10.1007/s11104-024-06853-x" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexswift2025drought()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractswift2025drought()">ABSTRACT</button>



<div id="aswift2025drought" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{swift2025drought,
  title = {Drought stress homogenizes maize growth responses to diverse natural soil microbiomes},
  author = {Swift, Joel F and Kolp, Matthew R and Carmichael, Amanda and Ford, Natalie E and Hansen, Paige M and Sikes, Benjamin A and Kleiner, Manuel and Wagner, Maggie R},
  journal = {Plant and Soil},
  volume = {509},
  number = {1},
  pages = {181--199},
  year = {2025},
  doi = {10.1007/s11104-024-06853-x}
}
</pre>
</div>


<div id="bswift2025drought" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>As the climate changes, plants and their associated microbiomes face greater water limitation and increased frequency of drought. Soil- and root-associated microbes perform important functions that affect plant drought resilience, but the dynamics of their interactions with the host are poorly understood. Plant growth responses to natural soil microbiomes, as well as patterns of root colonization by microbes, are challenging to predict. We collected soil microbiomes from four native prairies across a steep precipitation gradient in Kansas, USA. Seedlings of two Zea mays genotypes were inoculated with each soil microbiome in a factorial drought experiment. We investigated plant phenotypic and root microbiome responses to drought and compared the effects of natural microbiome variation on plant growth under water-limited and well-watered conditions. Drought caused plants to accumulate shoot mass more slowly and achieve greater root/shoot mass ratios. Drought restructured the bacterial root-associated microbiome via depletion of Pseudomonadota and enrichment of Actinomycetota, whereas the fungal microbiome was largely unaffected. Taxonomically distinct soil microbiomes from the four contrasting environments affected plant growth under well-watered but not drought conditions. We demonstrated that the functional consequences of naturally-occurring soil microbiome variation are dependent on water availability, suggesting that future drying climates may dampen plants’ responsiveness to beneficial and/or pathogenic microbes.</pre>
</div>

<script>
function toggleBibtexswift2025drought(parameter) {
    var x= document.getElementById('aswift2025drought');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractswift2025drought(parameter) {
    var x= document.getElementById('bswift2025drought');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="kelliher2025quantifying">Kelliher, J. M., Rodriguez, F. E., Johnson, L. Y. D., Roux, S., Smith, M., Clum, A., Lynch, W., Bias, C. H., Finks, S. S., Keenum, I., Kiledal, E. A., Lin, H.-A., Longley, R., McDonald, R., Pitot, T. M., Rodríguez-Ramos, J., Shen, J., Sprockett, D. D., Swift, J. F., … Eloe-Fadrosh, E. A. (2025). Quantifying the impact of workshops promoting microbiome data standards and data stewardship. <i>Scientific Reports</i>, <i>15</i>(1).</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="kelliher2025quantifying">Kelliher, J. M., Rodriguez, F. E., Johnson, L. Y. D., Roux, S., Smith, M., Clum, A., Lynch, W., Bias, C. H., Finks, S. S., Keenum, I., Kiledal, E. A., Lin, H.-A., Longley, R., McDonald, R., Pitot, T. M., Rodríguez-Ramos, J., Shen, J., Sprockett, D. D., Swift, J. F., … Eloe-Fadrosh, E. A. (2025). Quantifying the impact of workshops promoting microbiome data standards and data stewardship. <i>Scientific Reports</i>, <i>15</i>(1).</span></div> -->








<a href="http://doi.org/10.1038/s41598-025-89991-1" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexkelliher2025quantifying()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractkelliher2025quantifying()">ABSTRACT</button>



<div id="akelliher2025quantifying" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{kelliher2025quantifying,
  title = {Quantifying the impact of workshops promoting microbiome data standards and data stewardship},
  author = {Kelliher, Julia M and Rodriguez, Francisca E and Johnson, Leah Y D and Roux, Simon and Smith, Montana and Clum, Alicia and Lynch, Wendi and Bias, Candace Hope and Finks, Sarai S and Keenum, Ishi and Kiledal, E Anders and Lin, Heng-An and Longley, Reid and McDonald, Ryan and Pitot, Thomas M and Rodríguez-Ramos, Josué and Shen, Jiaxian and Sprockett, Daniel D and Swift, Joel F and Yadav, Archana and Eloe-Fadrosh, Emiley A},
  journal = {Scientific reports},
  volume = {15},
  number = {1},
  year = {2025},
  doi = {10.1038/s41598-025-89991-1}
}
</pre>
</div>


<div id="bkelliher2025quantifying" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>The field of microbiome research continues to grow at a rapid pace, with multi-omics approaches becoming widely used to interrogate diverse microbiome samples. However, due to lagging awareness and implementation of standards and data stewardship, many datasets are produced that are not comparable, reproducible, or reusable. In 2021, the National Microbiome Data Collaborative launched its Ambassador Program, which utilizes a community-learning model to annually train a cohort of early-career researchers in microbiome data stewardship best practices. These Ambassadors then host workshops and other events to communicate these themes to their respective microbiome research communities. To quantify the impact of this learning model for promoting awareness of and experience with microbiome data, we conducted a survey of workshop participants from events hosted by the 2023 Ambassador cohort. The 2023 cohort of 13 National Microbiome Data Collaborative Ambassadors collectively hosted 21 events, reaching over 550 researchers. The Ambassadors distributed an anonymous post-workshop survey to their event participants to quantify the effectiveness of the training materials, the workshop format, and the thematic content. From the 21 events, survey results were successfully collected for 15 of those events from a total of 122 researchers. Overall, 122 participants working with a range of microbiome types and from a variety of institutions responded to the survey and reported overwhelmingly positive experiences with the workshop content and materials, with 98% of respondents reporting that they gained knowledge from the event. Participants across the events also reported an increase in their post-workshop understanding of metadata standards, principles for microbiome data management and reporting, and the importance of standardization in microbiome data processing. Participants also expressed a willingness to apply what they learned about microbiome data stewardship to their own research. The results of this study demonstrate the effectiveness of hands-on workshops and community-learning for communicating data stewardship best practices to microbiome researchers. The lessons learned and details about the implementation of this cohort-based learning model contained herein are intended to assist other groups in their efforts to create or improve similar learning strategies.</pre>
</div>

<script>
function toggleBibtexkelliher2025quantifying(parameter) {
    var x= document.getElementById('akelliher2025quantifying');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractkelliher2025quantifying(parameter) {
    var x= document.getElementById('bkelliher2025quantifying');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="migicovsky2024grapevine">Migicovsky, Z., Swift, J. F., Helget, Z., Klein, L. L., Ly, A., Maimaitiyiming, M., Woodhouse, K., Fennell, A., Kwasniewski, M., Miller, A. J., &amp; others. (2024). Grapevine leaf size influences canopy temperature. <i>OENO One</i>, <i>58</i>(2).</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="migicovsky2024grapevine">Migicovsky, Z., Swift, J. F., Helget, Z., Klein, L. L., Ly, A., Maimaitiyiming, M., Woodhouse, K., Fennell, A., Kwasniewski, M., Miller, A. J., &amp; others. (2024). Grapevine leaf size influences canopy temperature. <i>OENO One</i>, <i>58</i>(2).</span></div> -->








<a href="http://doi.org/10.20870/oeno-one.2024.58.2.7735" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexmigicovsky2024grapevine()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractmigicovsky2024grapevine()">ABSTRACT</button>



<div id="amigicovsky2024grapevine" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{migicovsky2024grapevine,
  title = {Grapevine leaf size influences canopy temperature},
  author = {Migicovsky, Zoe and Swift, Joel F and Helget, Zachary and Klein, Laura L and Ly, Anh and Maimaitiyiming, Matthew and Woodhouse, Karoline and Fennell, Anne and Kwasniewski, Misha and Miller, Allison J and others},
  journal = {OENO One},
  volume = {58},
  number = {2},
  year = {2024},
  doi = {10.20870/oeno-one.2024.58.2.7735}
}
</pre>
</div>


<div id="bmigicovsky2024grapevine" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Grapevine leaves have diverse shapes and sizes which are influenced by many factors including genetics, vine phytosanitary status, environment, leaf and vine age, and node position on the shoot. To determine the relationship between grapevine leaf shape or size and leaf canopy temperature, we examined five seedling populations grown in a vineyard in California, USA. The populations had one parent with compound leaves of the Vitis piasezkii type and a different second parent with non-compound leaves. In previous work, we had measured the shape and size of the leaves collected from these populations using 21 homologous landmarks. Here, we paired these morphological data with canopy temperature measurements made using a handheld infrared thermometer. After recording time of sampling and canopy temperature, we used a linear model between time of sampling and canopy temperature to estimate temperature residuals. Based on these residuals, we determined if the canopy temperature of each vine was cooler or warmer than expected, based on the time of sampling. We established a relationship between leaf size and canopy temperature: vines with larger leaves were cooler than expected. By contrast, leaf shape was not strongly correlated with variation in canopy temperature. Ultimately, these findings indicate that vines with larger leaves may contribute to the reduction of overall canopy temperature; however, further work is needed to determine whether this is due to variation in leaf size, differences in the openness of the canopy or other related traits.</pre>
</div>

<script>
function toggleBibtexmigicovsky2024grapevine(parameter) {
    var x= document.getElementById('amigicovsky2024grapevine');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractmigicovsky2024grapevine(parameter) {
    var x= document.getElementById('bmigicovsky2024grapevine');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="swift2023grapevine">Swift, J. F., Migicovsky, Z., Trello, G. E., &amp; Miller, A. J. (2023). Grapevine bacterial communities display compartment-specific dynamics over space and time within the Central Valley of California. <i>Environmental Microbiome</i>, <i>18</i>(1).</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="swift2023grapevine">Swift, J. F., Migicovsky, Z., Trello, G. E., &amp; Miller, A. J. (2023). Grapevine bacterial communities display compartment-specific dynamics over space and time within the Central Valley of California. <i>Environmental Microbiome</i>, <i>18</i>(1).</span></div> -->








<a href="http://doi.org/10.1186/s40793-023-00539-0" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexswift2023grapevine()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractswift2023grapevine()">ABSTRACT</button>



<div id="aswift2023grapevine" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{swift2023grapevine,
  title = {Grapevine bacterial communities display compartment-specific dynamics over space and time within the Central Valley of California},
  author = {Swift, Joel F and Migicovsky, Zoe and Trello, Grace E and Miller, Allison J},
  journal = {Environmental Microbiome},
  volume = {18},
  number = {1},
  year = {2023},
  doi = {10.1186/s40793-023-00539-0}
}
</pre>
</div>


<div id="bswift2023grapevine" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Plant organs (compartments) host distinct microbiota which shift in response to variation in both development and climate. Grapevines are woody perennial crops that are clonally propagated and cultivated across vast geographic areas, and as such, their microbial communities may also reflect site-specific influences. These site-specific influences along with microbial differences across sites compose ‘terroir’, the environmental influence on wine produced in a given region. Commercial grapevines are typically composed of a genetically distinct root (rootstock) grafted to a shoot system (scion) which adds an additional layer of complexity via genome-to-genome interactions. To understand spatial and temporal patterns of bacterial diversity in grafted grapevines, we used 16S rRNA amplicon sequencing to quantify soil and compartment microbiota (berries, leaves, and roots) for grafted grapevines in commercial vineyards across three counties in the Central Valley of California over two successive growing seasons. Community composition revealed compartment-specific dynamics. Roots assembled site-specific bacterial communities that reflected rootstock genotype and environment influences, whereas bacterial communities of leaves and berries displayed associations with time. These results provide further evidence of a microbial terroir within the grapevine root systems but also reveal that the microbiota of above-ground compartments are only weakly associated with the local soil microbiome in the Central Valley of California.</pre>
</div>

<script>
function toggleBibtexswift2023grapevine(parameter) {
    var x= document.getElementById('aswift2023grapevine');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractswift2023grapevine(parameter) {
    var x= document.getElementById('bswift2023grapevine');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="migicovsky2023x">Migicovsky, Z., Quigley, M. Y., Mullins, J., Ali, T., Swift, J. F., Agasaveeran, A. R., Dougherty, J. D., Grant, B. M., Korkmaz, I., Malpeddi, M. R., McNichol, E. L., Sharp, A. W., Harris, J. L., Hopkins, D. R., Jordan, L. M., Kwasniewski, M. T., Striegler, R. K., Dowtin, A. L., Stotts, S., … Chitwood, D. H. (2023). X-ray imaging of 30 year old wine grape wood reveals cumulative impacts of rootstocks on scion secondary growth and Ravaz index. <i>Horticulture Research</i>, <i>10</i>(1).</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="migicovsky2023x">Migicovsky, Z., Quigley, M. Y., Mullins, J., Ali, T., Swift, J. F., Agasaveeran, A. R., Dougherty, J. D., Grant, B. M., Korkmaz, I., Malpeddi, M. R., McNichol, E. L., Sharp, A. W., Harris, J. L., Hopkins, D. R., Jordan, L. M., Kwasniewski, M. T., Striegler, R. K., Dowtin, A. L., Stotts, S., … Chitwood, D. H. (2023). X-ray imaging of 30 year old wine grape wood reveals cumulative impacts of rootstocks on scion secondary growth and Ravaz index. <i>Horticulture Research</i>, <i>10</i>(1).</span></div> -->








<a href="http://doi.org/10.1093/hr/uhac226" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexmigicovsky2023x()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractmigicovsky2023x()">ABSTRACT</button>



<div id="amigicovsky2023x" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{migicovsky2023x,
  title = {X-ray imaging of 30 year old wine grape wood reveals cumulative impacts of rootstocks on scion secondary growth and Ravaz index},
  author = {Migicovsky, Zoe and Quigley, Michelle Y and Mullins, Joey and Ali, Tahira and Swift, Joel F and Agasaveeran, Anita Rose and Dougherty, Joseph D and Grant, Brendan Michael and Korkmaz, Ilayda and Malpeddi, Maneesh Reddy and McNichol, Emily L and Sharp, Andrew W and Harris, Jackie L and Hopkins, Danielle R and Jordan, Lindsay M and Kwasniewski, Misha T and Striegler, R. Keith and Dowtin, Asia L and Stotts, Stephanie and Cousins, Peter and Chitwood, Daniel H},
  journal = {Horticulture Research},
  volume = {10},
  number = {1},
  year = {2023},
  doi = {10.1093/hr/uhac226}
}
</pre>
</div>


<div id="bmigicovsky2023x" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Annual rings from 30 year old vines in a California rootstock trial were measured to determine the effects of 15 different rootstocks on Chardonnay and Cabernet Sauvignon scions. Viticultural traits measuring vegetative growth, yield, berry quality, and nutrient uptake were collected at the beginning (1995 to 1999) and end (2017 to 2020) of the lifetime of a vineyard initially planted in 1991 and removed in 2021. X-ray Computed Tomography (CT) was used to measure ring widths in 103 vines. Ring width was modeled as a function of ring number using a negative exponential model. Early and late wood ring widths, cambium width, and scion trunk radius were correlated with 27 traits. Modeling of annual ring width shows that scions alter the width of the first rings but that rootstocks alter the decay of later rings, consistently shortening ring width throughout the lifetime of the vine. Ravaz index, juice pH, photosynthetic assimilation and transpiration rates, and instantaneous water use efficiency are correlated with scion trunk radius. Ultimately, our research indicates that rootstocks modulate secondary growth over years, altering physiology and agronomic traits. Rootstocks act in similar but distinct ways from climate to modulate ring width, which borrowing techniques from dendrochronology, can be used to monitor both genetic and environmental effects in woody perennial crop species.</pre>
</div>

<script>
function toggleBibtexmigicovsky2023x(parameter) {
    var x= document.getElementById('amigicovsky2023x');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractmigicovsky2023x(parameter) {
    var x= document.getElementById('bmigicovsky2023x');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="lance2022multifaceted">Lance, R. F., Guan, X., Swift, J. F., Edwards, C. E., Lindsay, D. L., &amp; Britzke, E. R. (2022). Multifaceted DNA metabarcoding of guano to uncover multiple classes of ecological data in two different bat communities. <i>Evolutionary Applications</i>, <i>15</i>(7), 1189–1200.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="lance2022multifaceted">Lance, R. F., Guan, X., Swift, J. F., Edwards, C. E., Lindsay, D. L., &amp; Britzke, E. R. (2022). Multifaceted DNA metabarcoding of guano to uncover multiple classes of ecological data in two different bat communities. <i>Evolutionary Applications</i>, <i>15</i>(7), 1189–1200.</span></div> -->








<a href="http://doi.org/10.1111/eva.13425" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexlance2022multifaceted()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractlance2022multifaceted()">ABSTRACT</button>



<div id="alance2022multifaceted" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{lance2022multifaceted,
  title = {Multifaceted DNA metabarcoding of guano to uncover multiple classes of ecological data in two different bat communities},
  author = {Lance, Richard F and Guan, Xin and Swift, Joel F and Edwards, Christine E and Lindsay, Denise L and Britzke, Eric R},
  journal = {Evolutionary Applications},
  volume = {15},
  number = {7},
  pages = {1189--1200},
  year = {2022},
  doi = {10.1111/eva.13425}
}
</pre>
</div>


<div id="blance2022multifaceted" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>DNA contained in animal scat provides a wealth of information about the animal, and DNA metabarcoding of scat collections can provide key information about animal populations and communities. Next-generation DNA sequencing technologies and DNA metabarcoding provide an efficient means for obtaining information available in scat samples. We used multifaceted DNA metabarcoding (MDM) of noninvasively collected bat guano pellets from a Myotis lucifugus colony on Fort Drum Military Installation, New York, USA, and from two mixed-species bat roosts on Fort Huachuca Military Installation, Arizona, USA, to identify attributes such as bat species composition, sex ratios, diet, and the presence of pathogens and parasites. We successfully identified bat species for nearly 98% of samples from Fort Drum and 90% of samples from Fort Huachuca, and identified the sex for 84% and 67% of samples from these same locations, respectively. Species and sex identification matched expectations based on prior censuses of bat populations utilizing those roosts, though samples from some species were more or less common than anticipated within Fort Huachuca roosts. Nearly 62% of guano samples from Fort Drum contained DNA from Pseudogymnoascus destructans, where bats with wing damage from White-nose Syndrome were commonly observed. Putative dietary items were detected in a majority of samples from insectivorous bats on Fort Drum (81%) and Fort Huachuca (63%). A minority of guano samples identified as the nectarivorous Leptonycteris yerbabuenae (28%) provided DNA sequences from putative forage plant species. Finally, DNA sequences from both putative ecto- and endoparasite taxa were detected in 35% and 56% of samples from Fort Drum and Fort Huachuca, respectively. This study demonstrates that the combination of noninvasive sampling, DNA metabarcoding, and sample and locus multiplexing provide a wide array of data that are otherwise difficult to obtain.</pre>
</div>

<script>
function toggleBibtexlance2022multifaceted(parameter) {
    var x= document.getElementById('alance2022multifaceted');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractlance2022multifaceted(parameter) {
    var x= document.getElementById('blance2022multifaceted');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="migicovsky2022increases">Migicovsky, Z., Swift, J. F., Helget, Z., Klein, L. L., Ly, A., Maimaitiyiming, M., Woodhouse, K., Fennell, A., Kwasniewski, M., Miller, A. J., Cousins, P., &amp; Chitwood, D. H. (2022). Increases in vein length compensate for leaf area lost to lobing in grapevine. <i>American Journal of Botany</i>, <i>109</i>(7), 1063–1073.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="migicovsky2022increases">Migicovsky, Z., Swift, J. F., Helget, Z., Klein, L. L., Ly, A., Maimaitiyiming, M., Woodhouse, K., Fennell, A., Kwasniewski, M., Miller, A. J., Cousins, P., &amp; Chitwood, D. H. (2022). Increases in vein length compensate for leaf area lost to lobing in grapevine. <i>American Journal of Botany</i>, <i>109</i>(7), 1063–1073.</span></div> -->








<a href="http://doi.org/10.1002/ajb2.16033" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexmigicovsky2022increases()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractmigicovsky2022increases()">ABSTRACT</button>



<div id="amigicovsky2022increases" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{migicovsky2022increases,
  title = {Increases in vein length compensate for leaf area lost to lobing in grapevine},
  author = {Migicovsky, Zoe and Swift, Joel F and Helget, Zachary and Klein, Laura L and Ly, Anh and Maimaitiyiming, Matthew and Woodhouse, Karoline and Fennell, Anne and Kwasniewski, Misha and Miller, Allison J and Cousins, Peter and Chitwood, Daniel H},
  journal = {American Journal of Botany},
  volume = {109},
  number = {7},
  pages = {1063--1073},
  year = {2022},
  doi = {10.1002/ajb2.16033}
}
</pre>
</div>


<div id="bmigicovsky2022increases" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Leaf lobing and leaf size vary considerably across and within species, including among grapevines (Vitis spp.), some of the best-studied leaves. We examined the relationship between leaf lobing and leaf area across grapevine populations that varied in extent of leaf lobing. We used homologous landmarking techniques to measure 2632 leaves across 2 years in 476 unique, genetically distinct grapevines from five biparental crosses that vary primarily in the extent of lobing. We determined to what extent leaf area explained variation in lobing, vein length, and vein to blade ratio. Although lobing was the primary source of variation in shape across the leaves we measured, leaf area varied only slightly as a function of lobing. Rather, leaf area increases as a function of total major vein length, total branching vein length, and vein to blade ratio. These relationships are stronger for more highly lobed leaves, with the residuals for each model differing as a function of distal lobing. For leaves with different extents of lobing but the same area, the more highly lobed leaves have longer veins and higher vein to blade ratios, allowing them to maintain similar leaf areas despite increased lobing. These findings show how more highly lobed leaves may compensate for what would otherwise result in a reduced leaf area, allowing for increased photosynthetic capacity through similar leaf size.</pre>
</div>

<script>
function toggleBibtexmigicovsky2022increases(parameter) {
    var x= document.getElementById('amigicovsky2022increases');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractmigicovsky2022increases(parameter) {
    var x= document.getElementById('bmigicovsky2022increases');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="harris2021multi">Harris, Z. N., Awale, M., Bhakta, N., Chitwood, D. H., Fennell, A., Frawley, E., Klein, L. L., Kovacs, L. G., Kwasniewski, M., Londo, J. P., Ma, Q., Migicovsky, Z., Swift, J. F., &amp; Miller, A. J. (2021). Multi‐dimensional leaf phenotypes reflect root system genotype in grafted grapevine over the growing season. <i>GigaScience</i>, <i>10</i>(12).</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="harris2021multi">Harris, Z. N., Awale, M., Bhakta, N., Chitwood, D. H., Fennell, A., Frawley, E., Klein, L. L., Kovacs, L. G., Kwasniewski, M., Londo, J. P., Ma, Q., Migicovsky, Z., Swift, J. F., &amp; Miller, A. J. (2021). Multi‐dimensional leaf phenotypes reflect root system genotype in grafted grapevine over the growing season. <i>GigaScience</i>, <i>10</i>(12).</span></div> -->








<a href="http://doi.org/10.1093/gigascience/giab087" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexharris2021multi()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractharris2021multi()">ABSTRACT</button>



<div id="aharris2021multi" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{harris2021multi,
  title = {Multi‐dimensional leaf phenotypes reflect root system genotype in grafted grapevine over the growing season},
  author = {Harris, Zachary N and Awale, Mani and Bhakta, Niyati and Chitwood, Daniel H and Fennell, Anne and Frawley, Emma and Klein, Laura L and Kovacs, Laszlo G and Kwasniewski, Misha and Londo, Jason P and Ma, Qin and Migicovsky, Zoe and Swift, Joel F and Miller, Allison J},
  journal = {GigaScience},
  volume = {10},
  number = {12},
  year = {2021},
  doi = {10.1093/gigascience/giab087}
}
</pre>
</div>


<div id="bharris2021multi" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Modern biological approaches generate volumes of multi-dimensional data, offering unprecedented opportunities to address biological questions previously beyond reach owing to small or subtle effects. A fundamental question in plant biology is the extent to which below-ground activity in the root system influences above-ground phenotypes expressed in the shoot system. Grafting, an ancient horticultural practice that fuses the root system of one individual (the rootstock) with the shoot system of a second, genetically distinct individual (the scion), is a powerful experimental system to understand below-ground effects on above-ground phenotypes. Previous studies on grafted grapevines have detected rootstock influence on scion phenotypes including physiology and berry chemistry. However, the extent of the rootstock’s influence on leaves, the photosynthetic engines of the vine, and how those effects change over the course of a growing season, are still largely unknown. Here, we investigate associations between rootstock genotype and shoot system phenotypes using 5 multi-dimensional leaf phenotyping modalities measured in a common grafted scion: ionomics, metabolomics, transcriptomics, morphometrics, and physiology. Rootstock influence is ubiquitous but subtle across modalities, with the strongest signature of rootstock observed in the leaf ionome. Moreover, we find that the extent of rootstock influence on scion phenotypes and patterns of phenomic covariation are highly dynamic across the season. These findings substantially expand previously identified patterns to demonstrate that rootstock influence on scion phenotypes is complex and dynamic and underscore that broad understanding necessitates volumes of multi-dimensional data previously unmet.</pre>
</div>

<script>
function toggleBibtexharris2021multi(parameter) {
    var x= document.getElementById('aharris2021multi');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractharris2021multi(parameter) {
    var x= document.getElementById('bharris2021multi');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="meyer2021understanding">Meyer, E. M., Swift, J. F., Bassüner, B., Smith, S. A., Menges, E. S., Oberle, B., &amp; Edwards, C. E. (2021). Understanding how an amphicarpic species with a mixed mating system responds to fire: a population genetic approach. <i>AoB Plants</i>, <i>13</i>(6), plab067.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="meyer2021understanding">Meyer, E. M., Swift, J. F., Bassüner, B., Smith, S. A., Menges, E. S., Oberle, B., &amp; Edwards, C. E. (2021). Understanding how an amphicarpic species with a mixed mating system responds to fire: a population genetic approach. <i>AoB Plants</i>, <i>13</i>(6), plab067.</span></div> -->








<a href="http://doi.org/10.1093/aobpla/plab067" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexmeyer2021understanding()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractmeyer2021understanding()">ABSTRACT</button>



<div id="ameyer2021understanding" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{meyer2021understanding,
  title = {Understanding how an amphicarpic species with a mixed mating system responds to fire: a population genetic approach},
  author = {Meyer, Elena M and Swift, Joel F and Bass{\"u}ner, Burgund and Smith, Stacy A and Menges, Eric S and Oberle, Brad and Edwards, Christine E},
  journal = {AoB Plants},
  volume = {13},
  number = {6},
  pages = {plab067},
  year = {2021},
  doi = {10.1093/aobpla/plab067}
}
</pre>
</div>


<div id="bmeyer2021understanding" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Amphicarpic plants produce both above-ground and below-ground seeds. Because below-ground seeds are protected in the soil and may maintain viability when above-ground conditions are stressful, they were proposed as an adaptation to recolonize a site after disturbance. However, whether below-ground seeds are the main colonizers after a disturbance remains unknown. Our goal was to understand whether recolonization by an amphicarpic species after fire was accomplished primarily through germination of seeds produced above-ground or below-ground. We investigated Polygala lewtonii, an amphicarpic, perennial species endemic to fire-prone Florida sandhill and scrub, where fire kills plants but subsequently increases recruitment and population sizes. Polygala lewtonii produces three flower types: above-ground chasmogamous flowers and above-ground and below-ground cleistogamous flowers, with previous research demonstrating chasmogamous flowers produce a much greater proportion of seeds than cleistogamous flowers. We quantified outcrossing in seeds produced by chasmogamous flowers to determine whether it differed from the 100 % self-fertilized below-ground seeds. Approximately 25 % of seeds from chasmogamous flowers showed evidence of cross-pollination. Assuming that chasmogamous flowers produce the majority of the above-ground seeds, as was shown previously, this indicates it is possible to differentiate between germination by above-ground versus below-ground seeds in post-fire colonization. We next compared genetic diversity, admixture, inbreeding and population genetic structure pre- and post-fire. If fire promoted germination of chasmogamous seeds, heterozygosity and admixture would increase, and genetic structure and inbreeding would decrease. Instead, inbreeding and genetic structure increased and admixture decreased, suggesting that the below-ground selfed seeds (with limited dispersal ability) increased their contribution to the population after fire, possibly because fire reduced above-ground seed viability. Additionally, new alleles not found previously in range-wide analyses emerged from the seed bank post-fire. These results suggest that amphicarpy is a powerful adaptation to preserve genetic variation, maintain adaptive potential and promote rapid post-fire colonization.</pre>
</div>

<script>
function toggleBibtexmeyer2021understanding(parameter) {
    var x= document.getElementById('ameyer2021understanding');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractmeyer2021understanding(parameter) {
    var x= document.getElementById('bmeyer2021understanding');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="mohn2021phylogeographic">Mohn, R. A., Oleas, N. H., Smith, A. B., Swift, J. F., Yatskievych, G. A., &amp; Edwards, C. E. (2021). The phylogeographic history of a range disjunction in eastern North America: the role of post-glacial expansion into newly suitable habitat. <i>American Journal of Botany</i>, <i>108</i>(6), 1042–1057.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="mohn2021phylogeographic">Mohn, R. A., Oleas, N. H., Smith, A. B., Swift, J. F., Yatskievych, G. A., &amp; Edwards, C. E. (2021). The phylogeographic history of a range disjunction in eastern North America: the role of post-glacial expansion into newly suitable habitat. <i>American Journal of Botany</i>, <i>108</i>(6), 1042–1057.</span></div> -->








<a href="http://doi.org/10.1002/ajb2.1686" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexmohn2021phylogeographic()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractmohn2021phylogeographic()">ABSTRACT</button>



<div id="amohn2021phylogeographic" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{mohn2021phylogeographic,
  title = {The phylogeographic history of a range disjunction in eastern North America: the role of post-glacial expansion into newly suitable habitat},
  author = {Mohn, Rebekah A and Oleas, Nora H and Smith, Adam B and Swift, Joel F and Yatskievych, George A and Edwards, Christine E},
  journal = {American Journal of Botany},
  volume = {108},
  number = {6},
  pages = {1042--1057},
  year = {2021},
  doi = {10.1002/ajb2.1686}
}
</pre>
</div>


<div id="bmohn2021phylogeographic" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>A disjunct distribution, where a species’ geographic range is discontinuous, can occur through vicariance or long-distance dispersal. Approximately 75 North American plant species exhibit a  650 km disjunction between the Ozark and Appalachian regions. This disjunction is attributed to biogeographic forces including: (1) Eocene–Oligocene vicariance by the formation of the Mississippi embayment; (2) Pleistocene vicariance from interglacial flooding; (3) post-Pleistocene northward colonization from separate glacial refugia; (4) Hypsithermal vicariance due to climate fluctuations; and (5) recent long-distance dispersal. We investigated which of these pathways most likely gave rise to the Appalachian-Ozark disjunction in Delphinium exaltatum. We genotyped populations of D. exaltatum from five Ozark and seven Appalachian localities, analyzed genetic structure, tested the order and timing of divergences using DIYABC, and conducted niche reconstructions up to 21,000 years before present (YBP). Populations fell into five main genetic clusters, i.e., a group in the central Appalachians, and four “lowland” groups. DIYABC analyses showed the central Appalachian and lowland lineages diverging 11,300 to17,000 YBP, and the lowland groups diverging 6800 to 10,900 YBP. Niche reconstructions showed that suitable climate for the central Appalachian lineage experienced large spatial discontinuity starting 14,000 YBP, such that divergence and persistence before this period is less plausible than divergence thereafter. Our results did not fully support any of the original hypotheses. Rather, the oldest divergence likely occurred after 13,500 YBP through expansion into newly opened habitat in the Appalachians. The Appalachian-Ozark disjunction likely resulted from northward dispersal of the lowland lineage as climate warmed during the Holocene.</pre>
</div>

<script>
function toggleBibtexmohn2021phylogeographic(parameter) {
    var x= document.getElementById('amohn2021phylogeographic');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractmohn2021phylogeographic(parameter) {
    var x= document.getElementById('bmohn2021phylogeographic');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="edwards2021conservation">Edwards, C. E., Tessier, B. C., Swift, J. F., Bassüner, B., Linan, A. G., Albrecht, M. A., &amp; Yatskievych, G. A. (2021). Conservation genetics of the threatened plant species Physaria filiformis (Missouri bladderpod) reveals strong genetic structure and a possible cryptic species. <i>PLoS One</i>, <i>16</i>(3).</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="edwards2021conservation">Edwards, C. E., Tessier, B. C., Swift, J. F., Bassüner, B., Linan, A. G., Albrecht, M. A., &amp; Yatskievych, G. A. (2021). Conservation genetics of the threatened plant species Physaria filiformis (Missouri bladderpod) reveals strong genetic structure and a possible cryptic species. <i>PLoS One</i>, <i>16</i>(3).</span></div> -->








<a href="http://doi.org/10.1371/journal.pone.0247586" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexedwards2021conservation()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractedwards2021conservation()">ABSTRACT</button>



<div id="aedwards2021conservation" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{edwards2021conservation,
  title = {Conservation genetics of the threatened plant species Physaria filiformis (Missouri bladderpod) reveals strong genetic structure and a possible cryptic species},
  author = {Edwards, Christine E and Tessier, Brooke C and Swift, Joel F and Bass{\"u}ner, Burgund and Linan, Alexander G and Albrecht, Matthew A and Yatskievych, George A},
  journal = {PLoS One},
  volume = {16},
  number = {3},
  year = {2021},
  doi = {10.1371/journal.pone.0247586}
}
</pre>
</div>


<div id="bedwards2021conservation" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Understanding genetic diversity and structure in a rare species is critical for prioritizing both in situ and ex situ conservation efforts. One such rare species is Physaria filiformis (Brassicaceae), a threatened, winter annual plant species. The species has a naturally fragmented distribution, occupying three different soil types spread across four disjunct geographical locations in Missouri and Arkansas. The goals of this study were to understand: (1) whether factors associated with fragmentation and small population size (i.e., inbreeding, genetic drift or genetic bottlenecks) have reduced levels of genetic diversity, (2) how genetic variation is structured and which factors have influenced genetic structure, and (3) how much extant genetic variation of P. filiformis is currently publicly protected and the implications for the development of conservation strategies to protect its genetic diversity. Using 16 microsatellite markers, we genotyped individuals from 20 populations of P. filiformis from across its geographical range and one population of Physaria gracilis for comparison and analyzed genetic diversity and structure. Populations of P. filiformis showed comparable levels of genetic diversity to its congener, except a single population in northwest Arkansas showed evidence of a genetic bottleneck and two populations in the Ouachita Mountains of Arkansas showed lower genetic variation, consistent with genetic drift. Populations showed isolation by distance, indicating that migration is geographically limited, and analyses of genetic structure grouped individuals into seven geographically structured genetic clusters, with geographic location/spatial separation showing a strong influence on genetic structure. At least one population is protected for all genetic clusters except one in north-central Arkansas, which should therefore be prioritized for protection. Populations in the Ouachita Mountains were genetically divergent from the rest of P. filiformis; future morphological analyses are needed to identify whether it merits recognition as a new, extremely rare specie</pre>
</div>

<script>
function toggleBibtexedwards2021conservation(parameter) {
    var x= document.getElementById('aedwards2021conservation');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractedwards2021conservation(parameter) {
    var x= document.getElementById('bedwards2021conservation');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="swift2021grapevine">Swift, J. F., Hall, M. E., Harris, Z. N., Kwasniewski, M. T., &amp; Miller, A. J. (2021). Grapevine microbiota reflect diversity among compartments and complex interactions within and among root and shoot systems. <i>Microorganisms</i>, <i>9</i>(1), 92.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="swift2021grapevine">Swift, J. F., Hall, M. E., Harris, Z. N., Kwasniewski, M. T., &amp; Miller, A. J. (2021). Grapevine microbiota reflect diversity among compartments and complex interactions within and among root and shoot systems. <i>Microorganisms</i>, <i>9</i>(1), 92.</span></div> -->








<a href="http://doi.org/10.3390/microorganisms9010092" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexswift2021grapevine()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractswift2021grapevine()">ABSTRACT</button>



<div id="aswift2021grapevine" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{swift2021grapevine,
  title = {Grapevine microbiota reflect diversity among compartments and complex interactions within and among root and shoot systems},
  author = {Swift, Joel F and Hall, Megan E and Harris, Zachary N and Kwasniewski, Misha T and Miller, Allison J},
  journal = {Microorganisms},
  volume = {9},
  number = {1},
  pages = {92},
  year = {2021},
  doi = {10.3390/microorganisms9010092}
}
</pre>
</div>


<div id="bswift2021grapevine" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Grafting connects root and shoot systems of distinct individuals, bringing microbial communities of different genotypes together in a single plant. How do root system and shoot system genotypes influence plant microbiota in grafted grapevines? To address this, we utilized clonal replicates of the grapevine ‘Chambourcin’, growing ungrafted and grafted to three different rootstocks in three irrigation treatments. Our objectives were to (1) characterize the microbiota (bacteria and fungi) of below-ground compartments (roots, adjacent soil) and above-ground compartments (leaves, berries), (2) determine how rootstock genotype, irrigation, and their interaction influences grapevine microbiota in different compartments, and (3) investigate abundance of microorganisms implicated in the late-season grapevine disease sour rot (Acetobacterales and Saccharomycetes). We found that plant compartment had the largest influence on microbial diversity. Neither rootstock genotype nor irrigation significantly influenced microbial diversity or composition. However, differential abundance of bacterial and fungal taxa varied as a function of rootstock and irrigation treatment; in particular, Acetobacterales and Saccharomycetes displayed higher relative abundance in berries of grapevines grafted to ‘1103P’ and ‘SO4’ rootstocks and varied across irrigation treatments. This study demonstrates that grapevine compartments retain distinct microbiota and identifies associations between rootstock genotypes, irrigation treatment, and the relative abundance of agriculturally relevant microorganisms in the berries.</pre>
</div>

<script>
function toggleBibtexswift2021grapevine(parameter) {
    var x= document.getElementById('aswift2021grapevine');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractswift2021grapevine(parameter) {
    var x= document.getElementById('bswift2021grapevine');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="edwards2019evaluating">Edwards, C. E., Swift, J. F., Lance, R. F., Minckley, T. A., &amp; Lindsay, D. L. (2019). Evaluating the efficacy of sample collection approaches and DNA metabarcoding for identifying the diversity of plants utilized by nectivorous bats. <i>Genome</i>, <i>62</i>(1), 19–29.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="edwards2019evaluating">Edwards, C. E., Swift, J. F., Lance, R. F., Minckley, T. A., &amp; Lindsay, D. L. (2019). Evaluating the efficacy of sample collection approaches and DNA metabarcoding for identifying the diversity of plants utilized by nectivorous bats. <i>Genome</i>, <i>62</i>(1), 19–29.</span></div> -->








<a href="http://doi.org/10.1139/gen-2018-0102" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexedwards2019evaluating()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractedwards2019evaluating()">ABSTRACT</button>



<div id="aedwards2019evaluating" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{edwards2019evaluating,
  title = {Evaluating the efficacy of sample collection approaches and DNA metabarcoding for identifying the diversity of plants utilized by nectivorous bats},
  author = {Edwards, Christine E and Swift, Joel F and Lance, Richard F and Minckley, Thomas A and Lindsay, Denise L},
  journal = {Genome},
  volume = {62},
  number = {1},
  pages = {19--29},
  year = {2019},
  doi = {10.1139/gen-2018-0102}
}
</pre>
</div>


<div id="bedwards2019evaluating" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>In this study, we evaluated the efficacy of sample collection approaches and DNA metabarcoding to identify plants utilized by nectivorous bats. Samples included guano collected from beneath bat roosts and pollen-swabs from bat fur, both of which were subjected to DNA metabarcoding and visual identification of pollen (microscopy) to measure plant diversity. Our objectives were to determine whether DNA metabarcoding could detect likely food plants of nectivorous bats, whether sample types would produce different estimates of plant diversity, and to compare results of DNA metabarcoding to visual identification. Visual identification found that 99% of pollen was from Agave, which is thought to be the bats’ main food source. The dominant taxon found by metabarcoding was also Agavoideae, but a broader diversity of plant species was also detected, many of which are likely “by-catch” from the broader environment. Metabarcoding outcomes differed between sample types, likely because pollen-swabs measured the plant species visited by bats and guano samples measured all items consumed in the bat’s diet, even those that were not pollen or nectar. Overall, metabarcoding is a powerful, high-throughput tool to understand bat ecology and species interactions, but careful analysis of results is necessary to derive accurate ecological conclusions.</pre>
</div>

<script>
function toggleBibtexedwards2019evaluating(parameter) {
    var x= document.getElementById('aedwards2019evaluating');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractedwards2019evaluating(parameter) {
    var x= document.getElementById('bedwards2019evaluating');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="swift2018multifaceted">Swift, J. F., Lance, R. F., Guan, X., Britzke, E. R., Lindsay, D. L., &amp; Edwards, C. E. (2018). Multifaceted DNA metabarcoding: Validation of a noninvasive, next-generation approach to studying bat populations. <i>Evolutionary Applications</i>, <i>11</i>(7), 1120–1138.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="swift2018multifaceted">Swift, J. F., Lance, R. F., Guan, X., Britzke, E. R., Lindsay, D. L., &amp; Edwards, C. E. (2018). Multifaceted DNA metabarcoding: Validation of a noninvasive, next-generation approach to studying bat populations. <i>Evolutionary Applications</i>, <i>11</i>(7), 1120–1138.</span></div> -->








<a href="http://doi.org/10.1111/eva.12644" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexswift2018multifaceted()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractswift2018multifaceted()">ABSTRACT</button>



<div id="aswift2018multifaceted" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{swift2018multifaceted,
  title = {Multifaceted DNA metabarcoding: Validation of a noninvasive, next-generation approach to studying bat populations},
  author = {Swift, Joel F and Lance, Richard F and Guan, Xin and Britzke, Eric R and Lindsay, Denise L and Edwards, Christine E},
  journal = {Evolutionary Applications},
  volume = {11},
  number = {7},
  pages = {1120--1138},
  year = {2018},
  doi = {10.1111/eva.12644}
}
</pre>
</div>


<div id="bswift2018multifaceted" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>As multiple species of bats are currently experiencing dramatic declines in populations due to white-nose syndrome (WNS) and other factors, conservation managers have an urgent need for data on the ecology and overall status of populations of once-common bat species. Standard approaches to obtain data on bat populations often involve capture and handling, requiring extensive expertise and unavoidably resulting in stress to the bats. New methods to rapidly obtain critical data are needed that minimize both the stress on bats and the spread of WNS. Guano provides a noninvasive source of DNA that includes information from the bat, but also dietary items, parasites, and pathogens. DNA metabarcoding is a high-throughput, DNA-based identification technique to assess the biodiversity of environmental or fecal samples. We investigated the use of multifaceted DNA metabarcoding (MDM), a technique combining next-generation DNA sequencing (NGS), DNA barcodes, and bioinformatic analysis, to simultaneously collect data on multiple parameters of interest (bat species composition, individual genotype, sex ratios, diet, parasites, and presence of WNS) from fecal samples using a single NGS run. We tested the accuracy of each MDM assay using samples in which these parameters were previously determined using conventional approaches. We found that assays for bat species identification, insect diet, parasite diversity, and genotype were both sensitive and accurate, the assay to detect WNS was highly sensitive but requires careful sample processing steps to ensure the reliability of results, while assays for nectivorous diet and sex showed lower sensitivity. MDM was able to quantify multiple data classes from fecal samples simultaneously, and results were consistent whether we included assays for a single data class or multiple data classes. Overall, MDM is a useful approach that employs noninvasive sampling and a customizable suite of assays to gain important and largely accurate information on bat ecology and population dynamics.</pre>
</div>

<script>
function toggleBibtexswift2018multifaceted(parameter) {
    var x= document.getElementById('aswift2018multifaceted');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractswift2018multifaceted(parameter) {
    var x= document.getElementById('bswift2018multifaceted');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="lindsay2018comparison">Lindsay, D. L., Swift, J. F., Lance, R. F., &amp; Edwards, C. E. (2018). A comparison of patterns of genetic structure in two co-occurring Agave species (Asparagaceae) that differ in the patchiness of their geographical distributions and cultivation histories. <i>Botanical Journal of the Linnean Society</i>, <i>186</i>(3), 361–373.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="lindsay2018comparison">Lindsay, D. L., Swift, J. F., Lance, R. F., &amp; Edwards, C. E. (2018). A comparison of patterns of genetic structure in two co-occurring Agave species (Asparagaceae) that differ in the patchiness of their geographical distributions and cultivation histories. <i>Botanical Journal of the Linnean Society</i>, <i>186</i>(3), 361–373.</span></div> -->








<a href="http://doi.org/10.1093/botlinnean/box099" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexlindsay2018comparison()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractlindsay2018comparison()">ABSTRACT</button>



<div id="alindsay2018comparison" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{lindsay2018comparison,
  title = {A comparison of patterns of genetic structure in two co-occurring Agave species (Asparagaceae) that differ in the patchiness of their geographical distributions and cultivation histories},
  author = {Lindsay, Denise L and Swift, Joel F and Lance, Richard F and Edwards, Christine E},
  journal = {Botanical Journal of the Linnean Society},
  volume = {186},
  number = {3},
  pages = {361--373},
  year = {2018},
  doi = {10.1093/botlinnean/box099}
}
</pre>
</div>


<div id="blindsay2018comparison" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Spatial genetic structure in plants can be affected by continuity of geographical distribution, cultivation history, pollination and seed dispersal mechanisms. Here, we examined the relative effects of these factors on the genetic structure of Agave palmeri and A. parryi, which share similar pollination syndromes and seed dispersal mechanisms but differ in the continuity of their distributions and cultivation history. Agave parryi has a patchy distribution and was cultivated by Native Americans, whereas A. palmeri has a more continuous distribution and no documented history of cultivation. We sampled populations of each species in their North American ranges, genotyped individuals with microsatellite loci and analysed patterns of genetic structure. No genetic structuring was detected in A. palmeri, but A. parryi formed three distinct genetic clusters, corresponding to geographical locations. One genetic cluster of A. parryi contained populations from lower elevations and had lower levels of genetic diversity, probably indicating anthropogenic origins. We conclude that continuity of distribution, pollinators and cultivation history all may affect patterns of genetic connectivity.</pre>
</div>

<script>
function toggleBibtexlindsay2018comparison(parameter) {
    var x= document.getElementById('alindsay2018comparison');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractlindsay2018comparison(parameter) {
    var x= document.getElementById('blindsay2018comparison');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li>
<li><style>
.btn{
    margin-bottom:5px;
    padding-top:0px;
    padding-bottom:0px;
    padding-left:15px;
    padding-right:15px;
    height:20px
}

pre{
    white-space: pre-wrap;  
    white-space: -moz-pre-wrap; 
    white-space: -pre-wrap; 
    white-space: -o-pre-wrap; 
    word-wrap: break-word; 
    width:100%; overflow-x:auto;
}
</style>


<div class="text-justify"><span id="swift2016analysis">Swift, J. F., Smith, S. A., Menges, E. S., Bassüner, B., &amp; Edwards, C. E. (2016). Analysis of mating system and genetic structure in the endangered, amphicarpic plant, Lewton’s polygala (Polygala lewtonii). <i>Conservation Genetics</i>, <i>17</i>(6), 1269–1284.</span></div>

<!-- You can use the below to make your name bold -->
<!-- <span id="swift2016analysis">Swift, J. F., Smith, S. A., Menges, E. S., Bassüner, B., &amp; Edwards, C. E. (2016). Analysis of mating system and genetic structure in the endangered, amphicarpic plant, Lewton’s polygala (Polygala lewtonii). <i>Conservation Genetics</i>, <i>17</i>(6), 1269–1284.</span></div> -->








<a href="http://doi.org/10.1007/s10592-016-0860-3" target="_blank"><button class="btn btn-primary btm-sm">DOI</button></a>




<button class="btn btn-danger btm-sm" onclick="toggleBibtexswift2016analysis()">BIB</button>



<button class="btn btn-warning btm-sm" onclick="toggleAbstractswift2016analysis()">ABSTRACT</button>



<div id="aswift2016analysis" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>@article{swift2016analysis,
  title = {Analysis of mating system and genetic structure in the endangered, amphicarpic plant, Lewton’s polygala (Polygala lewtonii)},
  author = {Swift, Joel F and Smith, Stacy A and Menges, Eric S and Bass{\"u}ner, Burgund and Edwards, Christine E},
  journal = {Conservation genetics},
  volume = {17},
  number = {6},
  pages = {1269--1284},
  year = {2016},
  doi = {10.1007/s10592-016-0860-3}
}
</pre>
</div>


<div id="bswift2016analysis" style="display: none; background-color:black; border-radius:5px; padding:10px; margin-bottom:20px;">
<pre>Polygala lewtonii is a federally endangered, amphicarpic plant with a mixed mating system and three types of flowers: (1) aboveground, chasmogamous flowers (i.e., open-pollinated; CH), (2) aboveground, cleistogamous flowers (i.e., closed, selfing; CL) and (3) CL flowers on belowground stems (amphicarpy). Aboveground seeds are ant-dispersed, whereas belowground seeds are spaced across the length of the rhizome. Here, we collected individuals of P. lewtonii at both range-wide and fine geographic scales and genotyped them at 11 microsatellite loci. We analyzed patterns of genetic diversity and structure to understand: (1) the predominant mating system (selfing or outcrossing), (2) the movement of pollen and seeds across the landscape, and (3) the optimal strategy to conserve the full range of genetic variation. P. lewtonii reproduces predominantly by selfing or bi-parental inbreeding, but reproduction occurred through each of the three flower types. Some individuals produced by selfing/inbreeding were tightly clustered spatially, and were likely produced either by belowground flowers or by aboveground flowers with limited seed dispersal. Other selfed/inbred individuals were spatially separated (maximum of 15 m), and were likely produced by aboveground flowers followed by seed dispersal by ants. Fine-scale patterns of genetic structure indicate that some gene flow is occurring among aboveground CH flowers but both pollen and outcrossed seeds are moving limited distances (maximum of 0.5 km). Because genetic variation is structured at a fine spatial scale, protecting many populations is necessary to fully conserve the genetic variation in P. lewtonii. Conservation seed banking, if accompanied by research on seed germination requirements, may also contribute to the effective protection of genetic variation in P. lewtonii.</pre>
</div>

<script>
function toggleBibtexswift2016analysis(parameter) {
    var x= document.getElementById('aswift2016analysis');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
function toggleAbstractswift2016analysis(parameter) {
    var x= document.getElementById('bswift2016analysis');
    if (x.style.display === 'none') {
        x.style.display = 'block';
    } else {
        x.style.display = 'none';
    }
}
</script>
</li></ol>
</div>

</div>

      </div>
    </div>

    <br/>
<section id="footer">
<div class="container-footer">
  <div class="panel-footer">
	  <div class="row">
		<div class="col-sm-4">
		    <h5>About</h5>	
            <p>Joel F Swift<br/> NSF Plant Genome Postdoctural Fellow<br/> University of Kansas
</p>
		</div>

		<div class="col-sm-4">
		    <h5>Contact</h5>	
            <p><a href="mailto:joel.swift@ku.edu" target="_blank"><i class="fa fa-envelope fa-1x"></i> Contact Joel via email</a> <br/>
</p>
		</div>

		<div class="col-sm-4">
		    <h5>Coordinates</h5>	
            <p>University of Kansas<br/> 1200 Sunnyside Ave<br/> Lawrence, KS 66045
</p>
		</div>
	  </div>

      <center><p>&copy 2025 Joel F Swift </p></center>
	</div>
  </div>
</div>

<script src="/JoelFSwift.github.io/assets/javascript/bootstrap/bootstrap.bundle.min.js"></script>


  </body>

</html>
